src/modules/Dashboard/pages/Applet/Applet.tsx

```diff
@@ -22,8 +22,11 @@ export const Applet = () => {
 
   useEffect(() => {
     if (!id) return;
-    const { getApplet } = applets.thunk;
-    dispatch(getApplet({ appletId: id }));
+
+    const appletId = id;
```

Comment:
you could use
`const { id: appeltId } = useParams()`
to avoid unnecessary variables creations

Reply:
done